drive/amazon/amazon.go

```diff
@@ -46,29 +47,30 @@ func NewClient(c drive.Config) (drive.Client, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &AmazonCloudDrive{
+	return &Drive{
```

Comment:
Ugh, their marketing and technical departments don't seem to agree, as I now notice the URL embeded in endpoint.go is:
https://drive.amazonaws.com/drive/v1

I still think it's likely to be helpful to disambiguate from Google Drive somewhere along the way if it's called CloudDrive.